tests/integrational/asyncio/test_publish.py

```diff
@@ -64,24 +64,24 @@ async def test_publish_mixed_via_get(event_loop):
 
 
 @pn_vcr.use_cassette(
-    'tests/integrational/fixtures/asyncio/publish/object_via_get.yaml',
+    'tests/integrational/fixtures/asyncio/publish/object_via_get.json', serializer='pn_json',
     filter_query_parameters=['uuid', 'seqn', 'pnsdk'],
     match_on=['method', 'scheme', 'host', 'port', 'object_in_path', 'query']
 )
 @pytest.mark.asyncio
 async def test_publish_object_via_get(event_loop):
-    pubnub = PubNubAsyncio(pnconf_copy(), custom_event_loop=event_loop)
+    pubnub = PubNubAsyncio(pnconf_env_copy(), custom_event_loop=event_loop)
     await asyncio.ensure_future(assert_success_publish_get(pubnub, {"name": "Alex", "online": True}))
 
     await pubnub.stop()
 
 
 @pn_vcr.use_cassette(
-    'tests/integrational/fixtures/asyncio/publish/mixed_via_post.yaml',
-    filter_query_parameters=['uuid', 'seqn', 'pnsdk'])
+    'tests/integrational/fixtures/asyncio/publish/mixed_via_post.json', serializer='pn_json',
+    filter_query_parameters=['uuid', 'seqn', 'pnsdk', 'l_pub'])
 @pytest.mark.asyncio
 async def test_publish_mixed_via_post(event_loop):
-    pubnub = PubNubAsyncio(pnconf_copy(), custom_event_loop=event_loop)
```

Comment:
What's the difference between `pnconf_copy` and `pnconf_env_copy`?

Reply:
`pnconf_env*` uses keyset from environmental variables instead of those hardcoded. I'm slowly cleaning up what was in the repo for a while now